src/components/base/BaseIcon.vue

```diff
@@ -1,7 +1,12 @@
-<script setup>
+<script setup lang="ts">
 import { getIcon } from '@/utils/icons';
 
-const props = defineProps(['name']);
+//const props = defineProps(['name']);
+export interface Props {
+  name: string
```

Comment:
Should I set the type of the name to be part of a list of names, or just let it string ?

Reply:
string should be enough